README.md

```diff
@@ -1,57 +1,468 @@
+
+
+```yml
+# server host (serve command) - flag --host
+host: "localhost"
+# server port (serve command) - flag --port
+port: 8081
+
+couchdb:
+  # CouchDB server url - flag --couchdb-url
+  url: http://localhost:5984
+  # CouchDB user - flag --couchdb-user
+  user: ''
+  # CouchDB password - flag --couchdb-password
+  password: ''
+  # CouchDB prefix for the registries databases - flag --couchdb-prefix
+  prefix: registry1
+
+# Path to the session secret file containing the master secret to generate
+# session token.
+#
+# Should be generated with the "gen-session-secret" command.
+session-secret: sessionsecret.key
+```
+
+Feel free to change it if some configurations change in your case (the couchdb user, the server parameters or the databases prefix for example).
+
+> __Notices :__
+> - Here the generated session token (step 1) is stored in the `sessionsecret.key` file of the working directory, this is so the value of the property `session-secret` at the end of the configuration file.
+> - By default, the local couchdb allow all admin accesses without creating users, so there is no user and password here. But if an admin user has been created, you have to use the properties `user` and `password` in the `couchdb` part to provide these informations.
+
+### 3) Run the registry to serve the apps
+
+Run the registry using this followed command (here our configuration file is a `cozy-registry.yml` in the current working directory):
+
+```shell
+cozy-apps-registry serve -c cozy-registry.yml
+```
+
+At this step, you should have a cozy-apps-registry running and ready for the development.
+
+If you runnig the registry for the first time you can see the next steps to create a new editor and to configure your local `cozy-stack` to run with this new registry.
+
+> The `-c` options is always mandatory if you want to specify a config file like here.
+
+### 4) Create an editor
+
+This step is required to submit new applications in the registry.
+In another terminal, run these commands to create an editor (the one that will submit the application(s)) and generate an access token:
+
+```shell
+cozy-apps-registry add-editor Cozy -c cozy-registry.yml
+cozy-apps-registry gen-token Cozy -c cozy-registry.yml
+```
+
+Here we go, you have now an registry ready and running on the 8081 port.
+
+> The `-c` options is always mandatory if you want to specify a config file like here.
+
+### 5) Configure `cozy-stack` with the registry
+
+On the `cozy-stack`, you have to add the new registry in the stack configuration file [`cozy.yaml`](https://github.com/cozy/cozy-stack/blob/master/docs/config.md#main-configuration-file) (registries property):
+
+```yml
+...
+registries:
+  default:
+    - http://localhost:8081/
+...
+```
+
+Now restart your local `cozy-stack` to take this new configuration in consideration (stop and run again `cozy-stack serve`) and you're ready to work with the `cozy-apps-registry`!
+
+## Publish your application on the registry
```

Comment:
I'd prefer if the `Publish your application on the registry` can be moved to a tutorial on https://github.com/cozy/cozy-docs-v3

Reply:
Agree and keep a link in this README to have an entry from here as well 👍